<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqRecordEventsDialog</class>
 <widget class="QDialog" name="pqRecordEventsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="lastEvent">
      <attribute name="icon">
       <iconset resource="Resources/QtTesting.qrc">
        <normaloff>:/Icons/info.png</normaloff>:/Icons/info.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Last event</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="eventWidgetLabel">
           <property name="toolTip">
            <string>Last widget recorded</string>
           </property>
           <property name="text">
            <string>Widget :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eventWidgetEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="eventArgumentLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Last argument recorded</string>
           </property>
           <property name="text">
            <string>Argument :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eventArgumentEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="eventCommandLabel">
           <property name="toolTip">
            <string>Last command recorded / property checked</string>
           </property>
           <property name="text">
            <string>Command/Property :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eventCommandEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addComment">
      <attribute name="icon">
       <iconset resource="Resources/QtTesting.qrc">
        <normaloff>:/Icons/plus.png</normaloff>:/Icons/plus.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Add comment</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="commentLabel">
           <property name="text">
            <string>Add your custom comment :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="commentTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Write the comment that you want to add</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="blockingCheckBox">
             <property name="toolTip">
              <string>Add option block to you comment</string>
             </property>
             <property name="text">
              <string>Blocking</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="commentAddButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Add your comment</string>
             </property>
             <property name="text">
              <string>Add to macro</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/QtTesting.qrc">
               <normaloff>:/Icons/plus.png</normaloff>:/Icons/plus.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLCDNumber" name="nbEvents">
       <property name="toolTip">
        <string>numbers of events recorded</string>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nbEventsLabel">
       <property name="text">
        <string>Events recorded</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>133</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="checkButton">
       <property name="toolTip">
        <string>Check default property of a widget instead of recording event</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources/QtTesting.qrc">
         <normaloff>:/Icons/check.png</normaloff>:/Icons/check.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recordPauseButton">
       <property name="toolTip">
        <string>Start/Pause record</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources/QtTesting.qrc">
         <normaloff>:/Icons/record.png</normaloff>
         <normalon>:/Icons/pause.png</normalon>:/Icons/record.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Stop record</string>
       </property>
       <property name="text">
        <string>Stop Recording</string>
       </property>
       <property name="icon">
        <iconset resource="Resources/QtTesting.qrc">
         <normaloff>:/Icons/playback-stop.png</normaloff>:/Icons/playback-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/QtTesting.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>pqRecordEventsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
